src/test/common/pop-and-hazardous-components/pops-and-hazardous-components-warning-tests.js

```diff
@@ -39,9 +39,22 @@ export function popsAndHazardousComponentWarningTests(
       const payload = {
         wasteItems: [
           {
-            [popsOrHazardousObjectProperty]: {
-              [containsPopsOrHazardousField]: false
-            }
+            [containsPopsOrHazardousField]: false,
+            [popsOrHazardousObjectProperty]: {}
```

Comment:
From the name of this test, perhaps `[popsOrHazardousObjectProperty]: undefined` would be better for this test. It would also differentiate it from the test below?